docs/manual/jgmenu.1.md

```diff
@@ -83,6 +83,14 @@ For example:
 :   The description field is parsed as pango markup, so `<`, `>`, and `&`
     need to be escaped as `&lt;`, `&gt;`, and `&amp;` respectively.
 
+`"`
+
+:   Unlike in most CSV-based flavours, a single quote `"` is not a special delimiter.
```

Comment:
"Single quote" is ambiguous: it could mean `'` or a *single instance* of `"` ("double quote").
I think in this case it is meant to be the latter, but usually refers to the former.

Reply:
Thanks. Great catch. Must have been half asleep when merging.

How about  #234 ?

Reply:
The ambiguity is resolved by explicitly quoting the quotation mark in the text.
Would it be better to say _a single quotation mark_?
I do not call an apostrophe _a single quote_ but maybe some people do 🤔

Reply:
Single quote is different to double quote in both `shell` script and English. The same character is used for apostrophe and single quote.

I think @johnraff was right to point out the potential ambiguity so that we could make it extra clear and leave no doubt.